packages/atomic/src/components/search/search-box-suggestions/atomic-search-box-recent-queries/atomic-search-box-recent-queries.tsx

```diff
@@ -156,7 +155,7 @@ export class AtomicSearchBoxRecentQueries {
   private renderItem(value: string): SearchBoxSuggestionElement {
     const query = this.bindings.searchBoxController.state.value;
     return {
-      key: `recent-${cleanUpString(value)}`,
+      key: randomID('recent'),
```

Comment:
this was causing issues when we had, for instance `dress` and `dress=`. it would select both!

Reply:
this will always re-render though, what if we were encoding the string instead of cleaning it. It would be a replaceby instead of a remove

Reply:
https://stackoverflow.com/questions/7616461/generate-a-hash-from-string-in-javascript

<img width="665" alt="Screen Shot 2022-06-23 at 1 19 07 PM" src="https://user-images.githubusercontent.com/4923043/175357446-70ed8b49-bda7-49f1-8cc6-f5fcedf8570d.png">

Reply:
I can definitely try that, but it only rerenders when we hover over a new result, no? like, as you change the results, the dom elements disappear anyway, so is there a problem with the id changing for the next render?

Reply:
I'll remove this from this PR anyway so we can merge the fix quicker if you guys approve